tests/deployment/test_utils.py

```diff
@@ -0,0 +1,13 @@
+import pytest
+
+from bentoml.deployment.utils import ensure_docker_available_or_raise
+from bentoml.exceptions import BentoMLMissingDependencyException, BentoMLException
+
+
+def test_ensure_docker_available_or_raise():
```

Comment:
could you explain this test?

Reply:
file renamed to test_deployment_utils. no longer apply here